Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReport.c

```diff
@@ -460,6 +460,13 @@ uintptr_t *bsg_kscrw_i_getBacktrace(
         return backtraceBuffer;
      }
 
+#if TARGET_OS_WATCH
+    // The C++ exception stack trace trick doesn't work on watchOS, so don't capture it.
```

Comment:
i.e. our `__cxa_throw` function is not called?

Reply:
That code is unchanged, so it will still attempt, and fail for reasons I haven't figured out yet.

A task for later is to see if this can be made to work again.